### PR TITLE
Add logic for applying with prefix

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: server/server.go [hunk 1]
```diff
@@ -144,3 +150,18 @@ func psmdb(vs *pbVersion.VersionResponse, deps Deps, req *pbVersion.ApplyRequest
 
 	return nil
 }
+
+func validate(req *pbVersion.ApplyRequest) error {
```
Comment (qJkee) [inline server/server.go:154]: i don't like the function name at all, because it's not validating incoming request which is missleading

**Review by delgod — APPROVED** [review]

File: server/server.go (hunk 1 above)
Comment (blez) [reply server/server.go]: I do not know good name
https://github.com/Percona-Lab/percona-version-service/pull/31/commits/19c9a2e7ca0fec8ad26113b4c75cd747b6f701d6
If you have any suggestions let me know

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: server/server.go (hunk 1 above)
Comment (qJkee) [reply server/server.go]: i think this one is good enough =)

**Review by qJkee — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Pull request merged**